go1.14/Makefile

```diff
@@ -1,4 +1,4 @@
-IMAGES := base main darwin arm mips ppc s390x
+IMAGES := base main amd64 darwin arm armel armhf mips ppc s390x
```

Comment:
Perhaps `arm` should be renamed to `arm64`.

Reply:
And `main` has effectively become the `i386` image.

Reply:
We'll have to update anything that uses the images to pull the right arch flavor so I guess if we're changing one (main -> amd64) then it won't be too much to rename arm and main...?

Reply:
yes, we have to change the arch we pass when we launch the images, I start to find where this is in beats but I did not found it yet, Do you known where I have to touch to change it.

Reply:
It's at https://github.com/elastic/beats/blob/ded2e9912bb669e896b9c8d013dfabdb759023bd/dev-tools/mage/crossbuild.go#L180-L206. Journalbeat's magefile has a custom `ImageSelectorFunc` in order to use a different image due to debian7 + systemd issues. It will need updated too.

Reply:
We should probably make the image location configurable in Beats to allow for easier testing of the staged images (e.g. `export BEATS_GOLANG_CROSSBUILD_IMAGE=docker.elastic.co/observability-ci/golang-crossbuild`)

`BeatsCrossBuildImage = EnvOr("BEATS_GOLANG_CROSSBUILD_IMAGE", "docker.elastic.co/beats-dev/golang-crossbuild")`

https://github.com/elastic/beats/blob/7580c72445ecc6e97cab6fb7bcffc8063abf105b/dev-tools/mage/settings.go#L46